readme.md

```diff
@@ -30,42 +37,62 @@ https://github.com/alibaba/fastjson | e05e9c5e4be580691cc55a59f3256595393203a1 |
+
 ## Adding notes
 
 To add more information about any test:
 
-* Open an issue in this repository where the title is of the format:
+* Open an issue for **Java** in this repository where the title is of the format:
   * Project URL,SHA Detected,Fully-Qualified Test Name (packageName.ClassName.methodName): If the notes are for a particular test.
   * Project URL,SHA Detected,Module Path: If the notes are for a particular module.
   * Project URL,SHA Detected: If the notes are for a particular project.
+* Open an issue for **Python** in this repository where the title is of the format:
+  * Project URL,SHA Detected,Pytest Test Name (PathToFile::TestClass::TestMethod or PathToFile::TestMethod): If the notes are for a particular test.
+  * Project URL,SHA Detected: If the notes are for a particular project.
 * Make sure to include any steps to reproduce, any relevant logs, or any relevant information about the test/module in the issue. You should attach relevant files (e.g., log output of NonDex or iDFlakies) by [attaching the files to the issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests). E.g., 
   * for tests detected by iDFlakies, you should include `flaky-list.json` and `original-order` located under the `.dtfixingtools` directory generated by iDFlakies. Note that GitHub requires that these files be of `.txt` format (e.g., upload `flaky-list.txt` and `original-order.txt`, respectively).
   * for tests detected by NonDex, you should include `nondexMode` and `nondexSeed` from the `.nondex/{testid}/config` file generated by NonDex in the comments of the issue. You may also include the whole `config` file instead of adding a comment.
 
 An example issue can be found [here](https://github.com/TestingResearchIllinois/idoft/issues/88).
 
 #### Example entry in the [pr-data.csv file](https://github.com/TestingResearchIllinois/pr-data/blob/main/pr-data.csv):
+For Java:
 
 Project URL | SHA Detected | Module Path | Fully-Qualified Test Name (packageName.ClassName.methodName) | Category | Status | PR Link | Notes
 ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ----------
 https://github.com/wso2/carbon-apimgt | a82213e40e7e6aa529341fdd1d1c3de776949e64 | components/apimgt/org.wso2.carbon.apimgt.rest.api.commons | org.wso2.carbon.apimgt.rest.api.commons.util.RestApiUtilTestCase.testConvertYmlToJson | ID | Skipped | | https://github.com/TestingResearchIllinois/flaky-test-dataset/issues/1
 
+For Python:
+We do not have an example yet. If someone wants to open an issue for a Python test, the only two differences from Java are that (1) Python tests do not have the column of `Module Path`, and (2) the `Test Name` is Pytest Test Name (PathToFile::TestClass::TestMethod or PathToFile::TestMethod).
```

Comment:
the `Test Name` is Pytest Test Name (PathToFile::TestClass::TestMethod or PathToFile::TestMethod) -> the `Test Name` is `Pytest Test Name (PathToFile::TestClass::TestMethod or PathToFile::TestMethod)` ??